src/Symfony/Component/Ldap/Security/User/LdapUser.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Ldap\Security\User;
```

Comment:
do you think the `User` sub-namespace is a necessity? i tend to believe it can be left out

Reply:
I agree on this one, there's not going to be much in here besides of the `LdapUser`